apps/hip/static/js/rightScrollNav.js

```diff
@@ -1,17 +1,17 @@
 export default function () {
   /* Use IntersectionObserver to determine when a section of the page is within
-   * the user's view; when it is, add a CSS class to its respective nav title.
+   * the user's view; when it is, add a CSS class to its respective anchor tag.
+   * Docs: https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API
+   *       https://codepen.io/mishunov/pen/opeRdL
    */
 
-  const pageSections = document.querySelectorAll("section.page-section-hip");
-  const config = {
-    threshold: [0, 0.25, 0.5, 0.75, 1]
-  };
+  const pageSections = document.querySelectorAll("div.nav-heading-hip");
+  const config = {};
 
   // Define the IntersectionObserver.
   let observer = new IntersectionObserver(function (entries, self) {
```

Comment:
I think adding the `options` argument while instantiating `IntersectionObserver` will do the trick. In particular, I think we need to play with `threshold`. https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#creating_an_intersection_observer